usaspending_api/agency/v2/views/program_activity_count.py

```diff
@@ -37,7 +37,12 @@ def get_program_activity_count(self):
                         final_of_fy=True,
                         treasury_account__funding_toptier_agency=self.toptier_agency,
                         submission__reporting_fiscal_year=self.fiscal_year,
-                    ).values("pk")
+                    )
+                    .exclude(
```

Comment:
Note to self, @kbard check this SQL out in the morning to ensure it doesn't express same odd behavior as the other query we already replaced.

Reply:
Does not exhibit odd behavior we saw in other query, likely because it's not part of an aggregation.  👍